content/en/Getting started/Pentest objectives/_index.md

```diff
@@ -38,3 +35,6 @@ We use the penetration testing methodologies listed on the page. If you want to
 about each methodology, navigate to the page associated with your asset.
 
 If you start at the top of the pentest objectives page, your next step is to specify a [target](./pentest-target).
+
+<!-- For content, see https://github.com/cobalthq/cobalt-product-public-docs/blob/main/layouts/shortcodes/invite-help.md -->
```

Comment:
Until the PR is merged, you'll see this file in the following directory: https://github.com/cobalthq/cobalt-product-public-docs/tree/DOCS-70_incorporate_5steps_article_from_zendesk/layouts/shortcodes